Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/TimeView/TimeViewPlugin.cpp

```diff
@@ -183,14 +184,33 @@ namespace EMStudio
         // Top-left
         m_timeInfoWidget = new TimeInfoWidget(this);
         m_timeInfoWidget->setFixedWidth(175);
+        // Top-right
+        auto rightSideToolbar = new QToolBar{};
+        rightSideToolbar->addAction(m_togglePresetsView);
         topLayout->addWidget(m_timeInfoWidget);
         topLayout->addWidget(m_timeViewToolBar);
+        topLayout->addWidget(rightSideToolbar);
         mainLayout->addLayout(topLayout, 0, 0, 1, 2);
-
+        //
+        // ┌─────────────────┬────────────────────────────────────┬───────────────────────┐
+        // │  timeInfoWidget │  timeViewToolBar                   │rightSideToolbar       │
+        // └─────────────────┴────────────────────────────────────┴───────────────────────┘
+        // ┌─────────────────────────────────────────────────┬────────────────────────────┐
+        // │                  contentContainer               │                            │
+        // │  ┌─────────────────────────────────────────┐    │                            │
+        // │  │            contentLayout                │    │                            │
+        // │  │ ┌─────────────────────────────────────┐ │    │                            │
+        // │  │ │         trackAndTrackDataLayout     │ │    │     motionEventsPresets    │
+        // │  │ ├─────────────────────────────────────┤ │    │                            │
+        // │  │ │                                     │ │Splitter                         │
+        // │  │ │          bodyWidget                 │ │    │                            │
+        // │  │ └─────────────────────────────────────┘ │    │                            │
+        // │  │                                         │    │                            │
+        // │  └─────────────────────────────────────────┘    │                            │
+        // └─────────────────────────────────────────────────┴────────────────────────────┘
```

Comment:
Nice!

Reply:
Haha yeah that's some impressive ascii art skills 😅